pkg/pillar/cmd/baseosmgr/handlebaseos.go

```diff
@@ -419,23 +419,11 @@ func validatePartition(ctx *baseOsMgrContext,
 	// version?
 	if otherPartState == "inprogress" &&
 		otherPartVersion == config.BaseOsVersion {
-
-		errStr := fmt.Sprintf("Attempt to reinstall failed update %s in %s: refused",
```

Comment:
Do we already set things string somewhere else?

Reply:
As part of baseosStatus for other partition, in "inprogress" state, we publish the error info, extracted from other partition reboot information.